### PR TITLE
chore: fix space in import statement

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

from 后面少空格。

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | --          |
| 🇨🇳 Chinese | --          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/builtins/ComponentMeta/index.tsx [hunk 1]
```diff
@@ -124,6 +124,7 @@ const ComponentMeta: React.FC<ComponentMetaProps> = (props) => {
     <span key="from" className={styles.from}>
       from
     </span>,
+    <span key="space"> </span>,
```
Comment (crazyair) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:127]:
```suggestion
    <span key="space" />,
```

File: .dumi/theme/builtins/ComponentMeta/index.tsx [hunk 2]
```diff
@@ -122,7 +122,7 @@ const ComponentMeta: React.FC<ComponentMetaProps> = (props) => {
     </span>,
     <span key="component" className={styles.component}>{`{ ${component} }`}</span>,
     <span key="from" className={styles.from}>
-      from
+      from{' '}
```
Comment (yoyo837) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:125]: Should write some comments here.

Comment (afc163) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:125]:
```suggestion
      {/* https://github.com/ant-design/ant-design/pull/48552 */}
      from{' '}
```

File: .dumi/theme/builtins/ComponentMeta/index.tsx [hunk 3]
```diff
@@ -122,7 +122,8 @@ const ComponentMeta: React.FC<ComponentMetaProps> = (props) => {
     </span>,
     <span key="component" className={styles.component}>{`{ ${component} }`}</span>,
     <span key="from" className={styles.from}>
-      from
+      {/* https://github.com/ant-design/ant-design/pull/48552 */}
+      from{' '}
```
Comment (crazyair) [inline .dumi/theme/builtins/ComponentMeta/index.tsx:126]:
```suggestion
      {'from '}
```

**Comment by afc163** [comment]
@Wxh16144 好主意。

**Review by li-jia-nan — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

**Pull request merged**